Lib/idlelib/config_key.py

```diff
@@ -219,33 +224,34 @@ def TranslateKey(self, key, modifiers):
         return key
 
     def OK(self, event=None):
-        if self.advanced or self.KeysOK():  # doesn't check advanced string yet
-            self.result=self.keyString.get()
+        key = self.keyString.get()
+        key.strip()
```

Comment:
The check for missing '>' is odd because that also can only happen with 'advanced' entry.  So either these are a holdover from when 'advanced' entries were run through this function, or they constitute a check on the components to string code that belongs in the test module.   There is still improvement possible.